installers/sasview.spec

```diff
@@ -4,25 +4,25 @@ import sys
 from pathlib import Path
 import warnings
 import platform
-import sys
+import sysconfig
 
 block_cipher = None
-PYTHON_LOC = sys.exec_prefix
+PYTHON_PACKAGES = sysconfig.get_path('platlib')
```

Comment:
Maybe `site_package_path = ...`? The name `PYTHON_PACKAGES` sounds like a list of package names. `PYTHON_PACKAGE_PATH` would also work.

Reply:
That might be a better name yes, but that was a local copy of #2281 that I needed to hold on to deal with platform specific goo; since #2281 has been merged it will actually disappear from this PR.